src/pages/AddPage/index.jsx

```diff
@@ -0,0 +1,216 @@
+import { useState, useRef, useEffect } from "react";
+
+import Card from "../../components/Card";
+import Button from "../../components/Button";
+import Header from "../../components/Header";
+import Form from "../../components/Form";
+import PasswordForm from "../../components/PasswordForm";
+
+import { PageWrapper, CardWrapper, FooterWrapper } from "./style";
+
+import { ReactComponent as Arrow } from '../../assets/arrow.svg';
+import { CARD_NUMBER, DUE_DATE, OWNER, CVC, PASSWORD, COLORS, MONTH } from '../../constant'
+
+function AddPage() {
+  const [cardNumbers, setCardNumbers] = useState(["", "", "", ""]);
+  const [dueDate, setDueDate] = useState(["", ""]);
+  const [owner, setOwner] = useState("");
+  const [cvc, setCvc] = useState("");
+  const [password, setPassword] = useState(["", ""]);
+  const [allRequired, setAllRequired] = useState(false);
+  const [error, setError] = useState([false, false]);
+
+
+  useEffect(() => {
+    setAllRequired(
+      cardNumbers.join("").length >= CARD_NUMBER.UNIT_LENGTH * 4 &&
```

Comment:
@.@ 넘어갈 일은 없겠지만, `같다`가 좀 더 정확한 조건이 아닐까요?

```suggestion
      cardNumbers.join("").length === CARD_NUMBER.UNIT_LENGTH * 4 &&
```

Reply:
습관적으로 `이상`을 사용했는데 의미적으로 `같다`가 맞는 것 같네요. 앞으로도 유의해서 사용해야겠어요!